ci/Dockerfile

```diff
@@ -8,20 +8,37 @@ ENV QTDIR="${QT_PATH}/${QT_VERSION}/${QT_PLATFORM}"
 ENV LD_LIBRARY_PATH="${QTDIR}/lib:${LD_LIBRARY_PATH}"
 # $OPENSSL_PREFIX is provided by the docker image
 ENV LIBRARY_PATH="${OPENSSL_PREFIX}/lib:${LIBRARY_PATH}"
+ENV OPENSSL_ROOT_DIR="${OPENSSL_PREFIX}"
 
 RUN export DEBIAN_FRONTEND=noninteractive \
  && sudo apt update -yq \
  && sudo apt install -yq software-properties-common \
  && sudo add-apt-repository -y ppa:git-core/ppa \
+ && sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test \
  && sudo apt update -yq \
  && sudo apt purge -yq gnupg \
  && sudo apt install -yq --fix-missing \
-      build-essential cmake jq git s3cmd gnupg2 ca-certificates \
+      jq git s3cmd gnupg2 ca-certificates \
+      build-essential extra-cmake-modules \
       libpcre3-dev libnss3 libxcomposite1 libxtst6 libpcsclite-dev \
       gstreamer1.0-plugins-good gstreamer1.0-plugins-bad \
       gstreamer1.0-plugins-ugly gstreamer1.0-libav \
       gstreamer1.0-tools gstreamer1.0-alsa gstreamer1.0-pulseaudio
 
+# C++17 is required
+RUN export DEBIAN_FRONTEND=noninteractive \
+ && sudo apt-get install -yq --fix-missing gcc-9 g++-9 \
+ && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 100 \
+ --slave /usr/bin/g++ g++ /usr/bin/g++-9
```

Comment:
Why not install that as part of the main `apt install` call?

Reply:
Simply wanted to keep things separated, this way it will not be need to jump across Dockerfile sections in order to remove this requirement (e.g. when the base image is updated to a newer version).

Nevertheless, I moved it as you suggested, to avoid duplication of this 'noninteractive' stuff.